modules/liveIntentIdSystem.js

```diff
@@ -227,6 +227,10 @@ export const liveIntentIdSubmodule = {
         result.sovrn = { 'id': value.sovrn, ext: { provider: LI_PROVIDER_DOMAIN } }
       }
 
+      if (value.adserver) {
+        result.adserver = { 'id': value.adserver, ext: { provider: LI_PROVIDER_DOMAIN } }
+      }
+
```

Comment:
That should change to `thetradedesk `. We can ask @deepthivenkat if the name used to expose the id in Prebid should be `thetradedesk` too.

Reply:
I changed it to `thetradedesk`

Reply:
would it be possible to make the provider as the name of the domain for which the id is going to be passed? Something like: 
let domain = window.location.hostname;
and then use source: domain

Reply:
It would be possible, but would require more work which could only be done beginning of the next year. Also, we would only be able to expose the current domain as is - e.g. if it is abc.nyt.com, it will be abc.nyt.com and not nyt.com as we would not know what is the top level domain to split by. There is also risk that it might not be accepted by Prebid.

Reply:
We can expose current domain as such and not origin apex domain. That is fine. Confirmed with prebid that changing provider to originating domain is ok.  The thought process is - the feature is to support open path and the publisher who wants to directly integrate with TD is the one considered as provider for the id

We should be changing ext.provider. Apologies for the confusion Wiem! I should have mentioned ext.provider in the example.

Reply:
Thank you @deepthivenkat for the clarification